CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   - CVE-2018-14719
   - CVE-2018-14718
   - CVE-2018-14721
+* Added archetype for project generation.
```

Comment:
Looks like this branch might need to be rebased on master to pickup the 3.2.5 release changes.

Reply:
👍